### ISSUE TITLE
Richer housing listing view to include summary information

### ISSUE THREAD

**Issue by philhawksworth** [philhawksworth/the-united-effort-orginization#2]
The listing view and search/filter results of housing should display some additional information about each housing item.

As seen here: https://ueo.netlify.app/housing/affordable-housing/

_Edited to match latest discussion:_

In addition to the Apartment/Building name, we should also indicate:

- waitlist status,
- unit types,
- address, 
- contact phone, 
- contact email, 
- website

**Comment by clairehubel** [comment]
OK to leave this page as including just the waitlist status and the address :)

**Comment by philhawksworth** [comment]
Question for @clairehubel — If somebody prints this page of search results, it will probably need to have a little more information on it in order to be useful. Should we include the phone number and other contact details too? Or do you think that just having this as a list to refer to without the contact details will be useful?

**Comment by clairehubel** [comment]
I agree that a little more info would be useful. I’d suggest including phone number, address, and price if possible, as well as waitlist status. Does that sound good?

**Comment by philhawksworth** [comment]
We should also ensure that we group units under a single item in the results as described by @trevorshannon in #13

**Comment by trevorshannon** [comment]
What is the current status of this issue?  I see number of bedrooms, waitlist status, city, and apartment name are already shown in the results list.

Is the plan to add in address to the results then, or just have that in the print version?

I'm happy to take this on, I just want to make sure I know what is to be done.  And also double-checking @philhawksworth is not on it already.

**Comment by trevorshannon** [comment]
Actually, I just spoke to @clairehubel, and this is the official request for the housing search results:

On the screen:
waitlist status, unit types, property name, address, contact phone, contact email, website

On a printed page:
Same as on the screen with the addition of the search criteria, date accessed, and UEO logo at top

**Comment by philhawksworth** [comment]
I did some preliminary work on this, but go for it, @trevorshannon! I'll assign it to you.

I suggest we track the implementation of the print style in another issue, and keep this focussed on what is displayed in the housing listing view. I've been doing various print view things.

**Comment by trevorshannon** [comment]
Added all data fields except contact email which is blocked by #31

**Comment by philhawksworth** [comment]
Adding the email if available and then done

**Comment by trevorshannon** [comment]
I have a branch for this already that is nearly done though admittedly it is a pretty simple change

**Comment by trevorshannon** [comment]
@philhawksworth I just published feat/add-email.  feel free to do a pull request or modify as you see fit.

**Comment by trevorshannon** [comment]
@philhawksworth it's pretty late by now across the pond, so I'll go ahead and do a pull request and merge for feat/add-email

Just let me know if this is not what you were aiming for and we can revert or modify as needed.
